qutebrowser/mainwindow/statusbar/tabindex.py

```diff
@@ -0,0 +1,42 @@
+# vim: ft=python fileencoding=utf-8 sts=4 sw=4 et:
+
+# Copyright 2014-2015 Florian Bruhin (The Compiler) <mail@qutebrowser.org>
```

Comment:
That should be 2015 only, and you might prefer your own name/mail here - but if you prefer, mine is okay as well.